src/libraries/System.Diagnostics.DiagnosticSource/tests/MetricEventSourceTests.cs

```diff
@@ -25,7 +25,638 @@ public MetricEventSourceTests(ITestOutputHelper output)
         }
 
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
-        [OuterLoop("Slow and has lots of console spew")]
+        //[OuterLoop("Slow and has lots of console spew")]
```

Comment:
For convenience I've been leaving these commenting as we iterate (and using a reduced IntervalSecs) - I'll revert all of this before merging.